ocaml/xapi-consts/api_messages.ml

```diff
@@ -365,3 +365,6 @@ let periodic_update_sync_failed = addMessage "PERIODIC_UPDATE_SYNC_FAILED" 3L
 
 let xapi_startup_blocked_as_version_higher_than_coordinator =
   addMessage "XAPI_STARTUP_BLOCKED_AS_VERSION_HIGHER_THAN_COORDINATOR" 2L
+
+let breach_on_vm_anti_affinity_rules =
+  addMessage "BREACH_ON_VM_ANTI_AFFINITY_RULES" 3L
```

Comment:
I think the `breach` here is not clear to the users.
How about to use its definition directly? `All VMs in the anti-affinity group are runing on a single host`?

Reply:
I totally agree with it!
but it`s defined at design, @gangj How do you think? I prefer to update the design

Reply:
I think the condition for the alert is "All running VMs in the anti-affinity group resident on a single host", and we are implementing this way, right? @LunfanZhang 
Sure, you can change the design if we have a better name,  I didn't got a good short one, is `all_running_vms_in_one_anti_affinity_grp_resident_in_one_host` acceptable? Otherwise I think we can add some comments here.

Reply:
The `all_running_vms_in_one_anti_affinity_grp_resident_in_one_host` is ok I.M.O